MapboxNavigation/NextBannerView.swift

```diff
@@ -70,4 +71,9 @@ open class NextBannerView: UIView {
         instructionLabel.rightAnchor.constraint(equalTo: rightAnchor, constant: -16).isActive = true
     }
     
+    func update(step: RouteStep, instruction: VisualInstruction) {
+        maneuverView.step = step
+        instructionLabel.instruction = instruction.primaryTextComponents
```

Comment:
An `update(list:of:properties:to:update:)` method is still a pretty raw wrapper on the previous approach of setting all the properties directly and individually. This is an opportunity for us to think about when RouteMapViewController updates these properties and why. That’ll naturally lead us to a more descriptive name for the method, so we don’t inadvertently call or neglect to call it in the future.

In this case, I noticed that InstructionsBannerView has separate methods for updating the step (as part of setting the route progress) and setting the instruction; why not follow the same pattern for NextBannerView?

Reply:
Agreed, noting that `maneuverView.step ` is used for deciding upon the which maneuver arrow to draw.

Reply:
Ideally, a developer could just give us a RouteProgress, and we'd be able to draw the correct UI for them.

Reply:
Good point. How about making as many of these classes as possible maintain a single RouteProgress-typed property? Each setter would take care of extracting the relevant information from that RouteProgress object. We could create a new protocol for classes that track state via a RouteProgress-typed property.